editor/editor_resource_picker.cpp

```diff
@@ -212,7 +253,7 @@ void EditorResourcePicker::_update_menu_items() {
 	}
 
 	// Add options to convert existing resource to another type of resource.
-	if (edited_resource.is_valid()) {
+	if (is_editable() && edited_resource.is_valid()) {
```

Comment:
Keep in mind if the base scene is editable (instantiated or inherited), even if the subresource is not editable you can still replace it by another custom one and this change will be kept.